authors/settings/prod.py

```diff
@@ -0,0 +1,14 @@
+import os
+import dj_database_url
+from authors.settings.base import *
+
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = False
+
+
+DATABASES = {
+    'default': dj_database_url.config(
+        default=config('DATABASE_URL')
+    )
+}
```

Comment:
Add End of File here.

Reply:
> Add End of File here.

I have added an end of file line in authors/settings/prod.py